app/controllers/quotes_controller.rb

```diff
@@ -45,6 +45,11 @@ def destroy
     redirect_to quotes_path
   end
 
+  def best
+    @quotes = Quote.all.sort_by(&:score).reverse
```

Comment:
Score is a method on the quote class though that sums all the votes attached to it so just doing that throws an ActiveRecord error.  

``` ruby
class Quote
  def score
    votes.sum(:value)
  end
end

I am certain there is a way to do this with just SQL that I would like to do/know but my SQL isn't there yet.